docusaurus/docs/components/analytics/analytics.md

```diff
@@ -46,69 +62,86 @@ entries.event: click
 entries.url: <url of the page where the user performed the action>
 ```
 
-#### Live example: [Storybook](https://availity.github.io/availity-react/storybook/?path=/story/components-analytics--default)
+NOTE: In the code sample above, the Reactstrap `<Button>` component is able forwarded these data attributes to the native `<button>` element that the component renders. Not all React components can accept these data analytics attributes. Use the Elements panel of DevTools to confirm that these attributes are added to the appropriate DOM element.
```

Comment:
"is able forwarded". I think there's a word missing in there?

Reply:
Fixed. Thanks!